Frontend/src/style/MyRecipes.css

```diff
@@ -102,7 +102,7 @@
     display: flex;
     flex-direction: column;
     position: relative;
-    background-image: url("../assets/fillElements/GreenCard256.png");
+    background-image: url("../assets/fillElements/InfoCard_Background.png");
     background-size: contain;
     background-repeat: no-repeat;
 
```

Comment:
Have you tried to style the div accordinlgy instead of using an image as background ?